client/components/tinymce/plugins/mentions/style.scss

```diff
@@ -0,0 +1,66 @@
+.suggestions {
+	width: 40%;
+
+	img {
```

Comment:
We prefer not to use element selectors and instead target elements with classes. You can read more about our CSS guidelines here: https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md